### PR TITLE
Feature/UserDefaults property wrapper

### PR DESCRIPTION
**Pull request by EgzonArifi**
- Unit tests
- Documentation

### CODE REVIEW

**Review by ArlindDushi — COMMENTED** [review]
I'd also suggest adding the possibility to remove the object for a key, what do you think? 
Other than that lgtm 👍

File: Sources/Core/Utilities/UserDefaults/UserDefault.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//
+//  UserDefault.swift
+//  
+//
+//  Created by Egzon Arifi on 25/01/2022.
+//
```
Comment (borut-t) [inline Sources/Core/Utilities/UserDefaults/UserDefault.swift:6]: Let's use a regular header with copyright
```swift
//
//  UserDefault.swift
//  PovioKit
//
//  Created by Egzon Arifi on 25/01/2022.
//  Copyright © 2022 Povio Inc. All rights reserved.
//
```

File: Tests/Tests/Utilities/UserDefaults/UserDefaultTests.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
+//
+//  UserDefaultTests.swift
+//  
+//
+//  Created by Egzon Arifi on 25/01/2022.
+//
```
Comment (borut-t) [inline Tests/Tests/Utilities/UserDefaults/UserDefaultTests.swift:6]: Let's use a regular header with copyright
```swift
//
//  UserDefault.swift
//  PovioKit
//
//  Created by Egzon Arifi on 25/01/2022.
//  Copyright © 2022 Povio Inc. All rights reserved.
//
```

**Review by borut-t — APPROVED** [review]
Great, I like it 👌

**Review by yllfejziu — APPROVED** [review]

**Comment by EgzonArifi** [comment]
> I'd also suggest adding the possibility to remove the object for a key, what do you think? 

@ArlindDushi I think that is not a responsibility of a property wrapper

**Pull request merged**